src/main/java/ev3dev/sensors/BaseSensor.java

```diff
@@ -15,6 +15,15 @@
 
     protected static final int SWITCH_DELAY = 400;
 
+    protected final String VALUE0 = "value0";
+    protected final String VALUE1 = "value1";
+    protected final String VALUE2 = "value2";
+    protected final String VALUE3 = "value3";
+    protected final String VALUE4 = "value4";
+    protected final String VALUE5 = "value5";
+    protected final String VALUE6 = "value6";
+    protected final String VALUE7 = "value7";
+
```

Comment:
Are these constants necessary? The diff on GitHub shows only a removal from IR sensor.

Reply:
Oh, I have looked it up. I think that the situation at the end of this file could be better solved by a for-loop.